modules/microshift-install-rpm-before.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// microshift/microshift-install-rpm.adoc
+
+:_content-type: CONCEPT
+[id="microshift-before-install-rpm-preparing_{context}"]
+= Before installing {product-title} from an RPM package
+
+{product-title} uses the logical volume manager storage (LVMS) Container Storage Interface (CSI) provider for storing persistent volumes (PVs). LVMS relies on Linux's logical volume manager (LVM) to dynamically manage the backing storage for PVs. For this reason, your machine must have an LVM volume group (VG) with unused space in which LVMS can create the LVs for your workload's PVs.
+
+To configure a volume group (VG) that allows LVMS to create the LVs for your workload's PVs, adjust your root volume's size during the installation of {op-system}. Adjusting your root volume's size provides free space for additional LVs created by LVMS at runtime.
+
+The default name of the VG is `rhel`. You can change the VG name during your {op-system} configuration and use it during your installation.
```

Comment:
@dhellmann Is that true for 4.12 and 4.13, both?

Reply:
Those new rules apply to 4.13 and later. We're not making changes to 4.12.

Reply:
@dhellmann This language all exists in both versions with the exception of:
 
"The default name of the VG is `rhel`. You can change the VG name during your {op-system} configuration and use it during your installation." 

That's all this PR really adds as far as content. We're not saying has to change the name, just that they can. That's what the bug was filed to add. Do we want to add the sentences, or close the bug as Won't Do?

Reply:
The default is to use the volume group present on the system, regardless of its name, so they don't need to worry about renaming. In the unlikely event that they have multiple volume groups, it chooses the one called "microshift" instead of "rhel" now, so that comment about "rhel" being the default is no longer accurate.

I think I would just delete line 13.

Reply:
@dhellmann As discussed, are we good for this PR for 4.12 only? If so, please lgtm.

Reply:
@dhellmann Deleted. Is this PR for 4.13 lgtm?